lua/patches/0001-Makefile.patch

```diff
@@ -0,0 +1,34 @@
+From b6406f44544bf6b7542d9f3c3e175bdac2419f24 Mon Sep 17 00:00:00 2001
```

Comment:
Please also add `-ldl` on the `lua-"$LUA_VERSION"/src/Makefile`, in the `LIBS` variable, next to `-lm` as a patch.
Depending on your `gcc` version it may fail without this.